### ISSUE TITLE
Font face case discrepancies, Accessible DFA vs. Dfa (and DfA ;)

### ISSUE THREAD

**Issue by danielweck** [readium/readium-css#35]
`AccessibleDFA` should be `AccessibleDfa`:

https://github.com/readium/readium-css/blob/edfdc2a9c213023d4094b54dd3b57ae53d4af5bf/css/dist/ReadiumCSS-after.css#L676

https://github.com/readium/readium-css/blob/edfdc2a9c213023d4094b54dd3b57ae53d4af5bf/css/dist/ReadiumCSS-before.css#L310

(the file itself is `AccessibleDfA`, but that's alright ;)

**Comment by JayPanoz** [comment]
Ah thanks, good catch. It’s been corrected on the alpha branch. 👍

**Comment by JayPanoz** [comment]
Yes. I can confirm it should be fixed in the master branch now. 

But maybe you want to align on the file itself i.e. `AccessibleDfA`?

**Comment by danielweck** [comment]
I'm in favour of consistency, so if you want you may indeed use **A**ccessible**D**f**A** everywhere (i.e. filename, font-face declaration + font-family CSS property)
:)

**Comment by JayPanoz** [comment]
It’s now fixed in the develop branch.
